<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encode</class>
 <widget class="QWidget" name="Encode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>620</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>620</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frBg{
background-image:url(:/main/img/bg.jpg);
}
QFrame#frWhite{
background-color:rgb(255,255,255,230);
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frWhite">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>850</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:STXihei, &quot;华文细黑&quot;, &quot;Microsoft YaHei&quot;, &quot;微软雅黑&quot;;
font-size:20px;
}
QPushButton{
border-radius:15px;
background:rgb(237,200,199)

}
QPushButton:hover{
background:rgb(255,208,204);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnEncodeCancel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>440</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEncodeInput">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>120</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEncodeOutput">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>230</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="probarEncode">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>350</y>
       <width>540</width>
       <height>40</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEncodeEn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>440</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>转码</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEncodeInputPath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>二进制文件路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelOutputPath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>输出路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEncodeWarning">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>350</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255,0,0);</string>
     </property>
     <property name="text">
      <string>路径不能为空！</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditInput">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>450</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>输入路径</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOutput">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>450</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>输入路径</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
